packages/components/psammead-most-read/src/Item/index.jsx

```diff
@@ -129,10 +129,10 @@ export const MostReadItemWrapper = ({ dir, children, maxTwoColumns }) => (
 MostReadItemWrapper.propTypes = {
   children: node.isRequired,
   dir: oneOf(['rtl', 'ltr']),
-  maxTwoColumns: bool,
+  columnLayout: string,
```

Comment:
we can change this to `columnLayout: oneOf(['oneColumn', 'twoColumn', 'multiColumn'])`